x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/jinaai/response/JinaAIRerankResponseEntity.java

```diff
@@ -77,80 +74,78 @@ public class JinaAIRerankResponseEntity {
      * @throws IOException if there is an error parsing the response
      */
     public static InferenceServiceResults fromResponse(HttpResult response) throws IOException {
```

Comment:
Would it be worth updating the javadoc for this method to include the new behaviour? If you do decide to update it,  putting `<pre>` tags around the JSON parts would be helpful, to make the javadoc more readable in the IDE. Right now, when mousing over the method name to show the javadoc, the JSON is formatted all on one line, which is very difficult to read.